Matches the OS locale. We need to determine if this patch should be sent
upstream. 

--- mozilla-esr38/modules/libpref/init/all.js.orig      2015-02-18 16:39:16.189950136 +0530
+++ mozilla-esr38/modules/libpref/init/all.js   2015-02-18 16:40:56.002524458 +0530
@@ -1461,7 +1461,7 @@ pref("intl.charset.detector",
 pref("intl.charset.fallback.override",      "");
 pref("intl.charset.fallback.tld",           true);
 pref("intl.ellipsis",                       "chrome://global-platform/locale/intl.properties");
-pref("intl.locale.matchOS",                 false);
+pref("intl.locale.matchOS",                 true);
 // fallback charset list for Unicode conversion (converting from Unicode)
 // currently used for mail send only to handle symbol characters (e.g Euro, trademark, smartquotes)
 // for ISO-8859-1
